kyo-scheduler-zio/jvm/src/test/scala/kyo/KyoSchedulerZIOAppTest.scala

```diff
@@ -0,0 +1,9 @@
+package kyo
+
+import zio.ZIO
+
+object KyoSchedulerZIOAppDefaultTest extends KyoSchedulerZIOAppDefault {
```

Comment:
Unfortunately `ZIOAPP.main` method invokes `exit(0)`, so the JVM will shutdown in SBT (when running this within a test).

We can run this via `Test / run` - I can update the SBT configuration to include this in `test` if you want @fwbrasil

Reply:
```
[error] Error during tests:
[error]         Forked test harness failed: java.io.EOFException
[error]         at java.base/java.io.ObjectInputStream$BlockDataInputStream.peekByte(ObjectInputStream.java:3232)
[error]         at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1713)
[error]         at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:540)
[error]         at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:498)
[error]         at sbt.React.react(ForkTests.scala:202)
[error]         at sbt.ForkTests$Acceptor$1$.run(ForkTests.scala:133)
[error]         at java.base/java.lang.Thread.run(Thread.java:1583)

```

The error

Reply:
I think this is ok!